src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -1202,15 +1202,50 @@ func getTempUrls(securityScopedUrls: [URL]) -> [URL] {
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
+### Upload to a specified bucket
+
+You can also perform an upload operation to a specific bucket by providing the `bucket` option. This option can either be a string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console.
+
+See [additional storage bucket usage](/[platform]/build-a-backend/storage/set-up-storage/#additional-storage-bucket-usage) for more information.
+
+```ts title="Provide bucket as a string"
+import { uploadData } from 'aws-amplify/storage';
+
+try {
+  const result = uploadData({
+    path: "album/2024/1.jpg",
+    data: file,
+    options: {
+      bucket: "assignedNameInAmplifyBackend"
+    }
+  }).result;
+} catch (error) {
+  console.log(`Error: ${error}`)
+}
+```
+```ts title="Provide bucket as an object"
+options: {
+  bucket: { 
+    bucketName: "bucket-name-from-console", 
+    region: "us-east-2" 
+  }
+}
+```
+
+</InlineFilter>
+
+<InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
+
 ### More upload options
 
-Option | Type | Description |
-| -- | -- | ----------- |
-| contentType | String | The default content-type header value of the file when downloading it. <br/><br/> Read more at [Content-Type documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) |
-| contentEncoding | String | The default content-encoding header value of the file when downloading it.<br/><br/> Read more at [Content-Encoding documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding) |
-| contentDisposition | String | Specifies presentational information for the object. <br/><br/> Read more at [Content-Disposition documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition) |
-| metadata | map\<String\> | A map of metadata to store with the object in S3. <br/><br/> Read more at [S3 metadata documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingMetadata.html#UserMetadata) |
-| useAccelerateEndpoint | boolean | Whether to use accelerate endpoint. <br/><br/> Read more at [Transfer Acceleration](/[platform]/build-a-backend/storage/upload-files/#transfer-acceleration) |
+Option | Type | Default | Description |
```

Comment:
most of this should have some sort of default somewhere not in options we should mention that here i think. lets revisit this bit tomorrow

Reply:
I'm not sure exactly what you mean here. Are you suggesting explicitly calling out that the options object does not need to be supplied at all and defaults will be applied to some extent regardless?

Reply:
Resolved offline.